pkgs/development/tools/backblaze-b2/default.nix

```diff
@@ -1,21 +1,22 @@
-{ fetchFromGitHub, lib, python3Packages }:
+{ fetchFromGitHub, lib, python3Packages, setuptools_scm }:
 
 python3Packages.buildPythonApplication rec {
   pname = "backblaze-b2";
-  version = "2.1.0";
+  version = "2.4.0";
 
-  src = fetchFromGitHub {
-    owner = "Backblaze";
-    repo = "B2_Command_Line_Tool";
-    rev = "v${version}";
-    sha256 = "1kkpvxqgh5pw4kr8lh5gy9d7960hv9zvajbjiqhj6xgykwbpbgmq";
+  src = python3Packages.fetchPypi {
+    inherit version;
+    pname = "b2";
+    sha256 = "sha256-nNQDdSjUolj3PjWRn1fPBAEtPlgeent2PxzHqwH1Z6s=";
   };
 
-  propagatedBuildInputs = with python3Packages; [
+  propagatedBuildInputs = (with python3Packages; [
     b2sdk
     class-registry
     phx-class-registry
     setuptools
+  ]) ++ [
+    setuptools_scm
```

Comment:
setuptools-scm belongs into nativeBuildInputs because it is not required at runtime.

Reply:
Is this resolved? If so please explain.